json-form-react/utilities/JsonFormValidator.js

```diff
@@ -110,4 +120,115 @@ export default class JsonFormValidator {
     return null;
   }
 
+  _validateEmail(value) {
+
+    const validRegex = /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/;
+
+    if (!value.match(validRegex)) {
+
+      return 'invaid email type';
+    }
+    return null;
+  }
+
+
+  _validateDate(value) {
+
+    let isValidDate = Date.parse(value);
+
+    if (isNaN(isValidDate)) {
+      return 'Invalid Date Format';
+    }
+    return null;
+  }
+
+
+  _validateLetterOnly(value) {
+
+    const regEx = /^[A-Za-z]+$/;
+    if (!value.match(regEx)) {
+
+      return 'Please Enter Alphabetic Letters Only';
+    }
+    return null;
+  }
+
+  _validateNumbersOnly(value) {
+
+    const numbers = /^[0-9]+$/;
+    if (!value.match(numbers)) {
+      return 'Please Enter Only Number';
+    }
+
+    return null;
+  }
+
+  _validateDate1(value) {
+    var dateformat = /^(0?[1-9]|[12][0-9]|3[01])[\/\-](0?[1-9]|1[012])[\/\-]\d{4}$/;
```

Comment:
@hariidev Can we be sure that always the date will come in this format from the `<input type="date">` ?

Reply:
Sorry @hariidev I just saw this function is not being used... so if it is not needed can we delete it and keep only `_validateDate()`?

Reply:
yes i can delete it.